boards/Kconfig

```diff
@@ -38,14 +40,7 @@ config NET_DRIVERS
 	  When building for a qemu target then NET_DRIVERS will be default
 	  enabled to allow for easy use of SLIP or PPP
 
-# Note: $BOARD_DIR might be a glob pattern
-
-choice
-	prompt "Board Selection"
-
-source "$(BOARD_DIR)/Kconfig.board"
-
-endchoice
+rsource "Kconfig.$(BOARD_SCHEME)"
```

Comment:
This makes it seem like we might have independently versioned "schemes" for both boards and socs.

That's not what I expect: I expect a single hardware model v2, with changes for both boards and socs.

So should this BOARD_SCHEME be named something like HW_MODEL?